platform/security/sts/security-sts-anonymousclaimshandler/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -25,8 +25,12 @@
         <cm:managed-properties persistent-id="ddf.security.sts.anonymousclaims"
                                update-strategy="container-managed"/>
         <!-- Default properties -->
-        <property name="attributes"
-                  value="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier=anonymous,http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=anonymous"/>
+        <property name="attributes">
+            <array value-type="java.lang.String">
+                <value>http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier=anonymous</value>
```

Comment:
This change is going to collide with https://github.com/codice/ddf/pull/331 so depending on which one goes in first, this is going to need to be fixed.